### ISSUE TITLE
VTK8 - rendering/transparency test is crashing in parallel,scalable,icet mode

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#767]
The rendering/transparency test is crashing in parallel,scalable,icet mode. The test has been added to the skip list. It should be removed from the skip list when this bug is fixed.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Eric Brugger
Original creation: 06/13/2018 12:12 pm
Original update: 06/13/2018 12:15 pm
Ticket number: 3161
